examples/mvt.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @module examples.main
```

Comment:
```suggestion
 * @module examples.mvt
```
But I don't know if the module tag is really useful